docs/content/manual/release-specific/v1.2.0/test-backing-image-upload.md

```diff
@@ -14,6 +14,24 @@ title: Test backing image
 9. Create and attach a volume with the backing image. Verify the data content is correct.
 10. Do cleanup.
 
+### Upload via Rancher URI
+<sup>Related issue: [3129](https://github.com/longhorn/longhorn/issues/3129)</sup>
+1. Create or import cluster into Rancher
+2. Go into the cluster
+3. Click Longhorn from the left sidebar
+4. Open Longhorn UI from the Rancher UI
+   - An example URL: <small>`https://rancer.server.domain/k8s/clusters/CLUSTER-ID/api/v1/namespaces/longhorn-system/services/http:longhorn-frontend:80/proxy/#/backingImage`</small>
+5. Create a new backing image with **Upload From Local**
+6. Verify the uploaded image matches its size or checksum
```

Comment:
Do we want to create a volume using this backing image to see if this is working fine?

Reply:
I think it's okay as the attaching part tested in main test case. Validating the checksum should be sufficient for making sure uploading image files behind the proxies.